quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -335,6 +349,12 @@ impl Limits {
     pub fn initial_round_trip_time(&self) -> Duration {
         self.initial_round_trip_time
     }
+
+    #[doc(hidden)]
+    #[inline]
+    pub fn active_migration_enabled(&self) -> bool {
+        matches!(self.migration_support, MigrationSupport::Enabled)
+    }
```

Comment:
Even though the docs are hidden this is still part of the public API. Since its only used in one place, should we just do the check manually there?

Reply:
This is in `s2n-quic-core` and the usage of it is in `s2n-quic-transport`. Current this field (and all the others in `Limits`) is `pub(crate)`, I'd have to make it just `pub` to use it. I'd prefer just to have the getter.